plugin-api/src/test/java/org/sonar/api/config/PropertyDefinitionTest.java

```diff
@@ -251,6 +251,23 @@ public void validate_regular_expression() {
     assertThat(def.validate("[a-zA-Z").getErrorKey()).isEqualTo("notRegexp");
   }
 
+  @Test
+  public void validate_email() {
+    PropertyDefinition def = PropertyDefinition.builder("foo").name("foo").type(PropertyType.EMAIL).build();
+
+    assertThat(def.validate(null).isValid()).isTrue();
+    assertThat(def.validate("").isValid()).isTrue();
+    assertThat(def.validate("   ").isValid()).isTrue();
```

Comment:
I find those cases a bit strange, is it normal that an empty string is considered a valid email? Do you have an extra check on SQ side?